### DISCUSSION TITLE
Field required: API Rest

### DISCUSSION THREAD

**Discussion by ricmancio** [pods-framework/pods#6782] [Ideas]
## Issue Overview
When I added a field as a required on my custom CPT, I can call relative API without send this field and I received http status 201. Maybe if the field is required I cannot add anything.

## Expected Behavior
I want that when I add my field as a required field on my custom CPT, I can insert my new CPT only if in the body of REST API POST is declared the new field otherwise I have to receive an error if I don't add this specific field into body

## Current Behavior
Now when I try to add my new CPT from REST API I can do it, without receive any errors despite missing required field.

## Steps to Reproduce (for bugs)

1. create a new CPT with pods plugin where there is a new required field 
2. enable REST API for new CPT
3. open postman or other client and call POST REST API for added new CPT
4. I can do it also if missing the new required field

## WordPress Environment
WordPress 5.0.3 running Twenty Nineteen theme.

**Comment by quasel** [comment]
I see the point, a thing to think about!
